src/shell/navigation.test.js

```diff
@@ -0,0 +1,67 @@
+import { within } from '@testing-library/dom'
+
+import { saveFileStubFactory } from '../test-utils/stubs/saveFileStubFactory'
+import { farmhandStub } from '../test-utils/stubs/farmhandStub'
+import { nextView, previousView } from '../test-utils/ui'
+
+describe('navigation', () => {
+  test('cycles forwards through the standard views', async () => {
+    await farmhandStub()
+    const [header] = document.getElementsByTagName('header')
+
+    for (const viewName of ['Shop', 'Field', 'Workshop', 'Home']) {
+      await nextView()
+      within(header).getByText(viewName)
+    }
```

Comment:
there is no `expect` in any of these tests in this file

Reply:
I personally prefer every test have an expect (and in my world, only one expect 😄) but I do know what you mean about how they would still work as tests this way. I guess I like the explicit expect for consistency's sake, but won't hold up the PR for this if it's your style not to include them.

Reply:
I've actually been having a similar conversation about this exact thing at work this week, and my coworkers also prefer to have explicit `expect`ations, as they are clearer and easier to follow. So, it seems my opinion is unpopular. 😂

I'm fine with explicit `expect`ations. I've added them in https://github.com/jeremyckahn/farmhand/commit/37e407d8117b4993d2e17cf968c54ee9cc6504aa!